frontend/src/Components/ReusableComponents/UserInfo.jsx

```diff
@@ -0,0 +1,121 @@
+                              <p className="mb-0">Placeholder</p>
+                           </li>
+                           <li className="list-group-item d-flex justify-content-between align-items-center p-3">
+                              <i className="fab fa-github fa-lg github-icon-color" />
+                              <p className="mb-0">Placeholder</p>
+                           </li>
+                           <li className="list-group-item d-flex justify-content-between align-items-center p-3">
+                              <i className="fab fa-twitter fa-lg twitter-icon-color" />
+                              <p className="mb-0">Placeholder</p>
+                           </li>
+                           <li className="list-group-item d-flex justify-content-between align-items-center p-3">
+                              <i className="fab fa-instagram fa-lg instagram-icon-color" />
+                              <p className="mb-0">Placeholder</p>
+                           </li>
+                           <li className="list-group-item d-flex justify-content-between align-items-center p-3">
+                              <i className="fab fa-facebook-f fa-lg facebook-icon-color" />
+                              <p className="mb-0">Placeholder</p>
+                           </li>
+                        </ul>
+                     </div>
+                  </div>
+               </div>
+               <div className="col-lg-8">
+                  <div className="card mb-4 shadow">
+                     <div className="card-body">
+                        <div className="row">
+                           <div className="col-sm-3">Name:</div>
+                           <div className="col-sm-9 text-mutted">{userData?.name}</div>
+                        </div>
+                        <hr />
+                        <div className="row">
+                           <div className="col-sm-3">Last Name:</div>
+                           <div className="col-sm-9 text-mutted">{userData?.surname}</div>
+                        </div>
+                        <hr />
+                        <div className="row">
+                           <div className="col-sm-3">Email:</div>
+                           <div className="col-sm-9 text-mutted">{userData?.email}</div>
+                        </div>
+                        <hr />
+                        <div className="row">
+                           <div className="col-sm-3">Phone Number:</div>
+                           <div className="col-sm-9 text-mutted">{userData?.phoneNumber}</div>
+                        </div>
+                        <hr />
+                        <div className="row">
+                           <div className="col-sm-3">Address:</div>
+                           <div className="col-sm-9 text-mutted">{userData?.address}</div>
+                        </div>
+                        <hr />
+                        <div className="row">
```

Comment:
A lot of repetition, make smaller components where it makes sense.

Reply:
Fixed, thanks.